pkg/agent/openflow/network_policy.go

```diff
@@ -802,11 +817,19 @@ func (c *policyRuleConjunction) calculateClauses(rule *types.PolicyRule, clnt *c
 	var isEgressRule = false
 	switch rule.Direction {
 	case v1beta1.DirectionOut:
-		ruleTable = clnt.pipeline[egressRuleTable]
+		if rule.Priority != nil {
```

Comment:
For readability, could we use a method instead? something like `rule.IsAntreaNetworkPolicy()`

Reply:
Done.